zerver/webhooks/bitbucket3/view.py

```diff
@@ -114,25 +123,124 @@ def repo_push_handler(payload: Dict[str, Any], branches: Optional[str]=None) ->
+                                                                        url=pr["links"]["self"][0]["href"],
+                                                                        title=pr["title"])
+
+def get_pr_reassigned_body(payload: Dict[str, Any], include_title: Optional[bool]) -> str:
+    pr = payload["pullRequest"]
+    assignees_string = get_assignees_string(pr)
+    if not assignees_string:
+        assignees_string = "no one"
+    if not include_title:
+        return PULL_REQUEST_REASSIGNED_TEMPLATE.format(user_name=payload["actor"]["name"],
+                                                       number=pr["id"],
+                                                       url=pr["links"]["self"][0]["href"],
+                                                       assignees=assignees_string)
+    return PULL_REQUEST_REASSIGNED_TEMPLATE_WITH_TITLE.format(user_name=payload["actor"]["name"],
+                                                              number=pr["id"],
+                                                              url=pr["links"]["self"][0]["href"],
+                                                              assignees=assignees_string,
+                                                              title=pr["title"])
+
+def pr_handler(payload: Dict[str, Any], action: str,
+               include_title: Optional[bool]=False) -> List[Dict[str, str]]:
+    pr = payload["pullRequest"]
+    subject = get_pr_subject(pr["toRef"]["repository"]["name"], type="PR", id=pr["id"],
+                             title=pr["title"])
+    if action in ["opened", "modified"]:
+        body = get_pr_opened_or_modified_body(payload, action, include_title)
+    elif action == "needs_work":
+        body = get_pr_needs_work_body(payload, include_title)
+    elif action == "reviewers_updated":
+        body = get_pr_reassigned_body(payload, include_title)
+    else:
+        body = get_simple_pr_body(payload, action, include_title)
+
+    return [{"subject": subject, "body": body}]
+
+def pr_comment_handler(payload: Dict[str, Any], action: str,
+                       include_title: Optional[bool]=False) -> List[Dict[str, str]]:
+    pr = payload["pullRequest"]
+    subject = get_pr_subject(pr["toRef"]["repository"]["name"], type="PR", id=pr["id"],
+                             title=pr["title"])
+    message = payload["comment"]["text"]
+    if action == "deleted their comment on":
+        message = "~~{message}~~".format(message=message)
+    body = get_pull_request_event_message(user_name=payload["actor"]["name"],
+                                          action=action,
+                                          url=pr["links"]["self"][0]["href"],
+                                          number=pr["id"],
+                                          message=message,
+                                          title=pr["title"] if include_title else None)
+
```

Comment:
Hmm.. this is minor but could you please break up arguments on to the next line with indentation?
```
body = get_pull_request_event_message(
    user_name=payload["actor"]["name"],
    action=action,
)
```

It is much easier to read that way.

Reply:
> It is much easier to read that way.

From personal opinion, I really find the former method easier to read. Is it alright if I keep the indentation this way? If it, however, violates the indentation standards for this project, or if you really find the former indentation method to be fairly harder to read, I'll make the change right away. :slightly_smiling_face:

Reply:
@Hypro999: Okay, so there are a couple of reasons I disagree:
* That type of formatting is actually harder to read because, in a lot of cases, it results in lines of code that are longer than 80 characters. From a readability perspective (and even though we do allow lines longer than 80 characters), unless there is a good reason for extra long lines of code (for e.g., long URLs or SQL queries that can't be broken up over multiple lines), we should stick to the 80 character limit because it makes it easier to have to files open side by side in an editor in split-screen mode.
* While we do allow lines longer than 80 characters, in integrations, I have always tried my best to stick to that limit in new code for multi-file editing purposes. The only time that limit can be violated is if there is really no way around the long character count.

That's why I would recommend following that convention, we already follow it in a lot of our integrations and not to mention, consistency is a good thing! :) Thanks!

Reply:
>That type of formatting is actually harder to read because, in a lot of cases, it results in lines of code that are longer than 80 characters.

Yup, can't argue with that, the line lengths with the former method do become an issue. I've made the changes. :)